x-pack/plugins/apm/public/components/app/correlations/index.tsx

```diff
@@ -99,49 +110,61 @@ export function Correlations() {
-                  >
-                    {label}
-                  </EuiTab>
-                ))}
-              </EuiTabs>
+              {hasActivePlatinumLicense && (
+                <EuiTabs style={{ marginBottom: '-25px' }}>
+                  {tabs.map(({ key, label }) => (
+                    <EuiTab
+                      key={key}
+                      isSelected={key === currentTab}
+                      onClick={() => {
+                        setCurrentTab(key);
+                      }}
+                    >
+                      {label}
+                    </EuiTab>
+                  ))}
+                </EuiTabs>
+              )}
             </EuiFlyoutHeader>
             <EuiFlyoutBody>
-              <CorrelationsMetricsLicenseCheck>
-                {urlParams.kuery ? (
-                  <>
-                    <EuiCallOut size="s">
-                      <span>
-                        {i18n.translate(
-                          'xpack.apm.correlations.filteringByLabel',
-                          { defaultMessage: 'Filtering by' }
-                        )}
-                      </span>
-                      <EuiCode>{urlParams.kuery}</EuiCode>
-                      <EuiLink
-                        href={createHref(history, { query: { kuery: '' } })}
-                      >
-                        <EuiButtonEmpty size="xs" iconType="cross">
+              {hasActivePlatinumLicense ? (
+                <>
+                  {urlParams.kuery && (
+                    <>
+                      <EuiCallOut size="s">
+                        <span>
                           {i18n.translate(
-                            'xpack.apm.correlations.clearFiltersLabel',
-                            { defaultMessage: 'Clear' }
+                            'xpack.apm.correlations.filteringByLabel',
+                            { defaultMessage: 'Filtering by' }
                           )}
-                        </EuiButtonEmpty>
-                      </EuiLink>
-                    </EuiCallOut>
-                    <EuiSpacer />
-                  </>
-                ) : null}
-
-                <TabContent onClose={() => setIsFlyoutVisible(false)} />
-              </CorrelationsMetricsLicenseCheck>
+                        </span>
+                        <EuiCode>{urlParams.kuery}</EuiCode>
+                        <EuiLink
+                          href={createHref(history, { query: { kuery: '' } })}
+                        >
+                          <EuiButtonEmpty size="xs" iconType="cross">
+                            {i18n.translate(
+                              'xpack.apm.correlations.clearFiltersLabel',
+                              { defaultMessage: 'Clear' }
+                            )}
+                          </EuiButtonEmpty>
+                        </EuiLink>
+                      </EuiCallOut>
+                      <EuiSpacer />
+                    </>
+                  )}
```

Comment:
nit: I think it's a bit difficult to read with the nested condition, I'd extract it to another component or function just to clear the return object.